content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -34,7 +34,7 @@ <h3>Kubernetes Deployments</h3>
                 master schedules mentioned application instances onto individual Nodes in the cluster.
                 </p>
 
-                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces it. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
+                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller replaces the instance on another Node in the cluster. <b>This provides a self-healing mechanism to address machine failure or maintenance.</b></p>
```

Comment:
> replaces the instance on another Node in the cluster.

replaces it with an instance on another node in the cluster.

clearer?

Reply:
Does : replaces the instance with an instance on another node in the cluster.
Work?